src/site/content/en/blog/how-to-use-local-https/index.md

```diff
@@ -0,0 +1,317 @@
+authors:
+  - maudn
+date: 2020-12-07
+updated: 2020-12-07
+hero: hero.jpg
+tags:
+  - blog
+  - security
+---
+
+<hr/>
+
+**Outline:**
+
+- [Running your site locally with HTTPS using mkcert (recommended)](<#running-your-site-locally-with-https-using-mkcert-(recommended)>)
+- [Running your site locally with HTTPS: other options](#running-your-site-locally-with-https:-other-options)
+
+<hr/>
+
+_In this article, statements about `localhost` are valid for `127.0.0.1` and `[::1]` as well, since they both describe the local computer address, also called "loopback address". Also, to keep things simple, the port number isn't specified._
+_So when you see `http://localhost`, read it as `http://localhost:{port}` or `http://127.0.0.1:{port}`._
+
+If your production website uses HTTPS, you want your local development site to behave **like an
+HTTPS site** (if your production website doesn't use HTTPS, [make it a priority to switch to HTTPS](/why-https-matters/)).
+Most of the time, you can trust `http://localhost` to behave **like an HTTPS site**. But [in some
+cases](/when-to-use-local-https), you need to run your site locally with HTTPS. **Let's take a look
+at how to do this.**
+
+**⏩ Are you looking for quick instructions, or have you been here before?
+Skip to the [Cheatsheet](/#using-mkcert:-cheatsheet).**
+
+## Running your site locally with HTTPS using mkcert (recommended)
+
+To use HTTPS with your local development site and access `https://localhost` or
+`https://mysite.example` (custom hostname), you need a [TLS
+certificate](https://en.wikipedia.org/wiki/Public_key_certificate#TLS/SSL_server_certificate). But
+browsers won't consider just any certificate valid: your certificate needs to be **signed** by an entity that
+is trusted by your browser, called a trusted **[certificate authority (CA)](https://en.wikipedia.org/wiki/Certificate_authority)**.
+
+So, what you can do is create a certificate and sign it with a CA that is **trusted locally** by your
+device and browser.
+
+One tool that helps you do this in a few commands is
+[mkcert](https://github.com/FiloSottile/mkcert). Here is how it works:
+
+<figure class="w-figure">
+  <img src="./mkcert.jpg" alt="How mkcert works: diagram">
+  <figcaption class="w-figcaption">If you open your locally running site in your browser using HTTPS, your browser will check the
+certificate of your local development server. Upon seeing that the certificate has been signed by
+the mkcert-generated certificate authority, the browser checks whether it's registered as a trusted
+certificate authority. mkcert is listed as a trusted authority, so your browser trusts the
+certificate and creates an HTTPS connection.</figcaption>
+</figure>
+
+**Note:** mkcert isn't the only way to create a TLS certificate for local development, but it's a good
+one. See [other options](#running-your-site-locally-with-https:-other-options).
+
+{% Aside %}
+Why we're recommending mkcert and similar tools:
```

Comment:
```suggestion
mkcert (and similar tools) provide several benefits:
```

Reply:
Removing the aside here for semantics (this is important info, not supplemental), but also for readability. Having three different colored call-outs in a row adds visual clutter.